src/webview/wkwebview/mod.rs

```diff
@@ -307,8 +307,12 @@ impl InnerWebView {
             // So we pass the event to menu if we have one
             extern "C" fn key_down(this: &mut Object, _: Sel, event: id) {
               unsafe {
-                let superclass: *const Class = msg_send![this, superclass];
-                let () = msg_send![super(this, &*superclass), keyDown: event];
+                let keycode: u16 = msg_send![event, keyCode];
+                // Arrow key should not send for text
```

Comment:
won't this disable `keydown` events for Arrow keys?

Reply:
when I press left arrow, I found the cursor moved left twice. This means there's another key handling outside of this keydown chain. It could be performKeyEquivelant or doCommandBySelector, but I haven't verified which one and key event handling on macOS is a mess already.

But I know it's probably better to improve comment here. What do you think it should be?

Reply:
Whatever you feel like, just make sure this event https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event is triggered for Arrow keys, otherwise this will be a regression.

Reply:
Yes, I can see keycode 37~40 being triggered.